### DISCUSSION TITLE
tmp variable for remap #16

### DISCUSSION THREAD

**Discussion by i-m-amit** [sepal-contrib/sdg_15.3.1#21] [General]
Did you try the approach mentioned in issue #16 ?
It returns a name error : `NameError: name 'lc_transition_climate_coef_temporary' is not defined`
For the time being, I am reverting the tmp variable to run the code.

**Comment by 12rambau** [comment]
I run into the same problem. I was too fast when I run my suggestion and indeed you need the tmp variable as it is self referenced in the second line. 

sorry for the bad insight
